android/app/src/main/java/org/fossasia/openevent/common/api/OpenEventAPI.java

```diff
@@ -73,4 +74,7 @@
     @GET("discount-codes")
     Observable<List<DiscountCode>> getDiscountCodes();
 
+    @GET("feedbacks?include=user")
```

Comment:
Create an issue to include ?size=0 for all list API calls

Reply:
And already add it here

Reply:
will do that